client/src/API/AuthMethods.ts

```diff
@@ -1,31 +1,67 @@
-import axios from "axios";
+import axios, { AxiosError } from "axios";
 import { LoginData, UserInfoData, URL } from "../constants";
+import { useHistory } from "react-router-dom";
+
+const history = useHistory();
 
 export const signUp = async (userInfo: UserInfoData) => {
   try {
-    await axios.post(`${URL}/auth/signUp`, {
-      email: userInfo.email,
-      password: userInfo.password,
-      first_name: userInfo.firstName,
-      last_name: userInfo.lastName,
-    });
-  } catch (e: any) {
-    // Will need to research more on how to better type the error
-    if (e.response.status > 399) {
-      alert(e.response.data.message[0]);
+    const response = await axios.post(
+      `${URL}/auth/signUp`,
+      {
+        email: userInfo.email,
+        password: userInfo.password,
+        first_name: userInfo.firstName,
+        last_name: userInfo.lastName,
+      },
+      {
+        withCredentials: true,
+        headers: {
+          Authorization: `Bearer ${sessionStorage.getItem("authToken")}`,
+        },
+      }
+    );
+    sessionStorage.setItem("authToken", response.data.accessToken);
+    if (response) {
+      history.push("/");
     }
+  } catch (e) {
+    const err = e as AxiosError;
+    console.log(err.response?.data?.message);
   }
 };
 
 export const logIn = async (userInfo: LoginData) => {
   try {
-    await axios.post(`${URL}/auth/logIn`, {
-      email: userInfo.email,
-      password: userInfo.password,
-    });
-  } catch (e: any) {
-    if (e.response.status > 399) {
-      alert(e.response.data.message[0]);
+    const response = await axios.post(
+      `${URL}/auth/logIn`,
+      {
+        email: userInfo.email,
+        password: userInfo.password,
+      },
+      {
+        withCredentials: true,
```

Comment:
Same as above, we don't need withCredentials, bearer token stuff here. Just make sure that the accesstoken is set to session storage here! Not in the sign up method.